assets/shop/js/_visa_mobile.js

```diff
@@ -24,6 +39,7 @@ document.addEventListener('DOMContentLoaded', () => {
       return;
     }
 
+    phoneNumber.value = intlPhoneNumber.getNumber().substring(1);
```

Comment:
Removes the `+` sign from the beginning of the string

Reply:
Is it necessary if you have a JS that is preventing adding this kind of sign? (line 16-20)

Reply:
`getNumber` returns the string with `+` sign; prevention doesn't allow to type any other sign that digit; it's setting a value, not typing